### ISSUE TITLE
Strange Peep behavior when entering Pirate Ship ride

### ISSUE THREAD

**Issue by Dragondraikk** [OpenRCT2/OpenRCT2#2041]
I've recently been playing a multiplayer game with a friend and as we built a Pirate Ship ride, we noticed that it was not behaving as expected.

When peeps enter the ride, rather than walking up to their seats and getting in they walk right out to the side of the ride, through terrain and other rides to some arbitrary location before warping into their seats.

On top of that, as the ride is going the graphics do not render properly, creating ghosts and cut off pieces.

When they exit their seats they once again appear in a position far off the ride and proceed to run to a position right below the exit before floating up into it at which point they finally leave the ride and behave normally again.

All other rides are unaffected, and it only seems to happen in this one save I have. I also occurs across all versions and vanilla RCT2 in exactly the same way.

[Here is the savefile](https://openrct.net/cc/share/18/Evergreen+Gardens.SV6)

[And here is a video of the issue.](https://youtu.be/tk4nzGqvhNA)

**Comment by Patrik356b** [comment]
There was a similar issue with the haunted house #1723 (fixed now afaik)
It might be related tho.

**Comment by IntelOrca** [comment]
Hmm, does this save have the same problem in single player?

**Comment by Dragondraikk** [comment]
Yes, it happens in multiplayer, singleplayer, as well as the steam version of RCT2 on that save

**Comment by HaasJona** [comment]
I can't load this save with either OpenRCT2 or the regular RCT2.

**Comment by sjoerddebruin** [comment]
The save file isn't available anymore. Is this still happening in recent builds?

**Comment by Gymnasiast** [comment]
No, I believe it has been fixed long ago.

**Comment by jensj12** [comment]
I have a save (from a multiplayer server, 27th May 2016, contains custom objects): [ORCT Group Park Multiplayer 2.txt](https://github.com/OpenRCT2/OpenRCT2/files/313809/ORCT.Group.Park.Multiplayer.2.txt)

Take a look at 'Wheel of Torture'. Guests walk to the other side of the park (walking through everything) before entering the ride. The ride doesn't display it's graphics properly, just like the OP.
